### ISSUE TITLE
[Request] add property "+#% total to all Elemental Resistances" when price checking rare items as it was before in older version of APT

### ISSUE THREAD

**Issue by qisqz** [SnosMe/awakened-poe-trade#865]

**Comment by SnosMe** [comment]
add why?

**Comment by qisqz** [comment]
> add why?
i agree, that res choosing property not very useful thing, but choose "+#% total to all Elemental Resistances"  is must have feature
for example
some item with 120 total res and 40 "+#% total to all Elemental Resistances" probably worth more than same item with 48% first two resistance and 24 of last one
and u cant use harves if item already had 3 type of resistances
